applications/vanilla/modules/class.tagmodule.php

```diff
@@ -157,7 +157,7 @@ public function assetTarget() {
      * @return string
      */
     public function inlineDisplay() {
-        if (!c('EnabledPlugins.Tagging')) {
+        if (!c('Vanilla.Tagging.EnableUI')) {
```

Comment:
Enabled is misleading. Tagging is always "enabled" now. The UI component is what is disabled/enabled.

`Vanilla.Tagging.UI.Enabled`?